src/redux/actions/instructorActions.js

```diff
@@ -104,3 +105,24 @@ export const addNewProgram = programs => {
 export const setError = error => {
   return { type: SET_ERROR };
 };
+
+export const getNewsFeeds = (config) =>dispatch => {
+  try {
+    axios.get(
+      'https://coder-heroes-api.herokuapp.com/news', config
```

Comment:
We should be using our environment variables and our local databases for development rather than hardcoding our deployed back-end URL into our project like this. If this were using our ENV, then when deployed it would automatically use our front-end deployment's environment variable which points to our deployed back-end, unlike our locals in development -- and without exposing our back-end URL to anyone who knows how to "inspect" webpages!